charts/timescaledb-single/values.yaml

```
@@ -227,13 +227,21 @@ loadBalancer:
 
     # service.beta.kubernetes.io/aws-load-balancer-type: nlb            # Use an NLB instead of ELB
     # service.beta.kubernetes.io/aws-load-balancer-internal: 0.0.0.0/0  # Internal Load Balancer
+  # Define extra things that should go in the service spec
+  extraSpec: {}
+    # loadBalancerSourceRanges:
+    # - "0.0.0.0/0"
 
 replicaLoadBalancer:
   # If not enabled, we still expose the replica's using a so called Headless Service
   # https://kubernetes.io/docs/concepts/services-networking/service/#headless-services
   enabled: False
   annotations:
     service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout: "4000"
+  # Define extra things that should go in the service spec
+  extraSpec: {}
```

I'm wondering if we should just do, and merge the defaults of the template with the user specified values.
That would make it both flexible and work well out of the box.
```suggestion
  spec: {}
```

---

I've changed `extraSpec` to `spec`. Are you suggesting we take the stuff that is already in the service spec now and place that into the values.yaml so that it can be changed?